sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Utf8JsonWriterExtensions.cs

```diff
@@ -0,0 +1,103 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text.Json;
+using Azure.Core;
+
+namespace Azure.AI.MetricsAdvisor
+{
+    internal static class Utf8JsonWriterExtensions
+    {
+        public static void WriteNullStringValue(this Utf8JsonWriter writer, string propertyName, string value)
```

Comment:
I wonder if to simplify all this code, you could do something like this https://github.com/Azure/autorest.csharp/blob/f790582c450f6f90e173a12e96c8e83a745a0b0f/src/assets/Generator.Shared/Utf8JsonWriterExtensions.cs#L52

Reply:
I really liked the suggestion. Created an issue to track it (https://github.com/Azure/azure-sdk-for-net/issues/22472). Won't update the code now since it's only an internal change and it may affect 4 open PRs right now.